vhdbuilder/packer/install-dependencies.sh

```diff
@@ -438,11 +443,19 @@ PATCHED_HYPERKUBE_IMAGES="
 1.17.5.1
 1.18.1.1
 1.18.2.1
+1.19.0-beta.1
 "
 for KUBERNETES_VERSION in ${PATCHED_HYPERKUBE_IMAGES}; do
-  CONTAINER_IMAGE="mcr.microsoft.com/oss/kubernetes/hyperkube:v${KUBERNETES_VERSION}"
-  pullContainerImage "docker" ${CONTAINER_IMAGE}
-  echo "  - ${CONTAINER_IMAGE}" >> ${VHD_LOGS_FILEPATH}
+  if (($(echo ${KUBERNETES_VERSION} | cut -d"." -f2) < 17)); then
```

Comment:
before we move to use kube-proxy image in CCP, we still need hyperkube image for 1.17/1.18

Reply:
does that mean we should do x<1.19 use kyperkube instead of x<1.17

Reply:
Talked offline, so will use hyperkube for 1.17 and 1.18 as well